tensorflow_recommenders_addons/dynamic_embedding/python/keras/layers/embedding_test.py

```diff
@@ -68,6 +68,9 @@ def test_create(self):
 
     dims = [1, 4]
     key_dtypes = [dtypes.int32, dtypes.int64]
+    if test_util.is_gpu_available():
+      key_dtypes = [dtypes.int64]
```

Comment:
The int64 is already been introduced in [commit](https://github.com/tensorflow/recommenders-addons/commit/c2e897d6539eb8246f895ee03b4779cd9b929c92)